crates/editor/src/editor_tests.rs

```diff
@@ -11373,6 +11484,7 @@ pub fn handle_completion_request(
 
     let mut request = cx.handle_request::<lsp::request::Completion, _, _>(move |url, params, _| {
         let completions = completions.clone();
+        counter.fetch_add(1, atomic::Ordering::Acquire);
```

Comment:
```suggestion
        counter.fetch_add(1, atomic::Ordering::Release);
```

It's not important in this case I guess, but semantically correct to use it either this way, or we can use `SeqCst` everywhere, as it's just tests.